sbt/src/sbt-test/compiler-project/dotty-compiler-plugin/plugin/DivideZero.scala

```diff
@@ -10,31 +10,32 @@ import transform.MegaPhase.MiniPhase
 import Decorators._
 import Symbols.Symbol
 import Constants.Constant
-import transform.{LinkAll, Pickler}
+import transform.{Pickler, Staging}
 
+/** Compiler plugin that emits an error when compiling a division by zero */
 class DivideZero extends PluginPhase with StandardPlugin {
   val name: String = "divideZero"
   override val description: String = "divide zero check"
 
   val phaseName = name
 
   override val runsAfter = Set(Pickler.name)
-  override val runsBefore = Set(LinkAll.name)
+  override val runsBefore = Set(Staging.name)
```

Comment:
`Staging` is before `Pickler` -- so we need to change the ordering.

Reply:
I copy-pasted that file from https://github.com/lampepfl/dotty/blob/d78b8c0c1b34ecbbd052acd41e911daced1a95a1/sbt-dotty/sbt-test/sbt-dotty/compiler-plugin/plugin/DivideZero.scala#L23 so something must be wrong with the way we run our tests then :)